<!DOCTYPE html>
<html>
  <head>
    <meta name="pypi:repository-version" content="1.2">
    <title>Links for odoo13-addon-auth-from-http-remote-user</title>
  </head>
  <body>
    <h1>Links for odoo13-addon-auth-from-http-remote-user</h1>
<a href="https://files.pythonhosted.org/packages/63/78/9f9d5294791fdce496e22dc4052f5a5dc18fd3a10d4a63cbdf11f91c3be5/odoo13_addon_auth_from_http_remote_user-13.0.1.0.1.dev1-py3-none-any.whl#sha256=9bbc06056dd82bef7869b10ca4ef290f54cd7d84a078cd4817f2606384cc89ff" data-requires-python="&gt;=3.5" data-dist-info-metadata="sha256=05fe7dbbdf4e663cafdeb6761c578c8a46074de4f41ce41ea2aab4472d425203" data-core-metadata="sha256=05fe7dbbdf4e663cafdeb6761c578c8a46074de4f41ce41ea2aab4472d425203">odoo13_addon_auth_from_http_remote_user-13.0.1.0.1.dev1-py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/14/95/290b5ec83ea8d1591471f229b5db83b0fb6d921f97d470948a3111c386dd/odoo13_addon_auth_from_http_remote_user-13.0.1.0.1.dev2-py3-none-any.whl#sha256=5b0fd6b906b3735fe1b34edadd35ffc8ac4a7bfbb3052af4514051dd7b197395" data-requires-python="&gt;=3.5" data-dist-info-metadata="sha256=f8aad16fb10fd52e7a47c79626c285cce108a6633526f9e7833d506566c105ce" data-core-metadata="sha256=f8aad16fb10fd52e7a47c79626c285cce108a6633526f9e7833d506566c105ce">odoo13_addon_auth_from_http_remote_user-13.0.1.0.1.dev2-py3-none-any.whl</a><br />
</body>
</html>
<!--SERIAL 7415255-->